src/app/vendors/Vendor.tsx

```diff
@@ -43,7 +38,7 @@ export default function Vendor({
   ).join(', ');
 
   const vendorEngramDrops = $featureFlags.vendorEngrams
-    ? getVendorDropsForVendor(vendor.def.hash, allVendorEngramDrops)
+    ? store.getState().vendorDrops.vendorDrops.filter((vd) => vd.vendorId === vendor.def.hash)
```

Comment:
You should not directly access the store like this - it won't cause updates to the component when the store changes. Either pass this info in as a prop, or connect this component and fish it out of the store using `mapStateToProps`.

Reply:
Passing it in now.